tsdb/exemplar_test.go

```diff
@@ -442,6 +960,91 @@ func BenchmarkAddExemplar(b *testing.B) {
 	}
 }
 
+func BenchmarkAddExemplar_OutOfOrder(b *testing.B) {
```

Comment:
when running the benchamrks it's best to run them at least with `-count=6`. Otherwise `benchstat` can't give us statistically significant comparisons (it comaplins when you run it: `need >= 6 samples for confidence interval at level 0.95`)